sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -316,6 +320,17 @@ def _from_connection_string(
         return cls(**constructor_args)
 
     def _create_handler(self, auth: Union["pyamqp_JWTTokenAuthAsync", "uamqp_JWTTokenAuthAsync"]) -> None:
+        link_properties = {CONSUMER_IDENTIFIER: self._name}
+        if self._session_id == NEXT_AVAILABLE_SESSION:
+            timeout_in_ms = self._max_wait_time * 1000 if self._max_wait_time else 0
```

Comment:
does passing in 0 mean that the service will default to 65 secs? also, what does link_properties[OPERATION_TIMEOUT] = -100 mean in that case

Reply:
I realized that math mistake too, yes it will default to 65, I have moved this logic so it won't get set if it is not passed in, will wait to merge after tests are run :)